types/queries/insert_all_chapters.sql

```diff
@@ -0,0 +1,14 @@
+insert into collection_chapter (
+    title, 
+    document_id, 
+    wordpress_id, 
+    index_in_parent,
+    chapter_path)
+values (
+    $1, 
+    (select id
+    from document
+    where title = $2), 
```

Comment:
You can simplify this pattern by inserting the result of a select directly:
```sql
insert into ... (...)
select $1, document.id, $3, $4, $5
from document
where title = $2
```

by the way, you don't want to use `document.title`, you want to use `document.short_name` because that's `WJ36` instead of `Story of the Switch Striker` for example.

Reply:
Hmm, I tried rewriting it the way you suggested, but that meant that only chapters with an associated document ended up being inserted. And I can change it so that we're using the document short name-- the reason I had it the other way was since in the draft spreadsheet, the short name was marked as optional to include, and it was the long title that would be required if there was a document associated.

Maybe I misunderstood, in which case I can change it so it's reflected as short name required/longer title optional in the spreadsheet as well?

Reply:
Aha you are right about the optionality of the linked document, so yeah what you wrote seems best!

I repeat, we should remove the full document title from the spreadsheet entirely. The chapter itself has a title already and we don't need to repeat the document title on this table.

Reply:
Okay cool, I have removed the full document title from the spreadsheet.